scripts/zephyr_module.py

```diff
@@ -174,6 +174,15 @@
             type: seq
             sequence:
               - type: str
+  runners:
```

Comment:
Not sure if this should be put under a parent `west` mapping?

Reply:
I think a `west` mapping makes sense.
Especially `runners` is such an open word that it's not self-explaining that it relates to `west`.

To open a related discussion, what should be the guideline for putting something under `west` mapping ?
Because `runners` is not directly a west extension command, but used by the flash, debug, etc commands.
But we also have `blobs` and `package-manager` where a west extension command exists, but the information described by those can be valuable for projects not using west.

Reply:
I think in this case, `runners` are imported in `west_commands/run_common.py` so it is tightly coupled with `west`, no?